app/models/concerns/taxonomix.rb

```diff
@@ -132,6 +132,44 @@ def scope_by_taxable_ids(scope)
         scope.where("#{table_name}.id IN (#{cached_ids.join(',')})")
       end
     end
+
+    def join_organizations
+      join_taxonomies(Organization.all, org_join_arel)
```

Comment:
```suggestion
      join_taxonomies(Organization.unscoped, org_join_arel)
```
I think, `unscoped` is required here.

Reply:
Inside `join_taxonomies` method it requires only `taxonomy{location/organization}_ids`. So, would it be good to pass directly ids?

Reply:
Or probably just taxonomy class ?

Reply:
I think doing "unscoped" is optional or trivial since the only default scope is order(:title). Referring to https://github.com/theforeman/foreman/blob/develop/app/models/taxonomies/organization.rb#L22 unscoped is not used.

Reply:
I think I will prefer passing the scope to the method as i don't have to call pluck on every call manually.

Reply:
I'm for keeping the scope, as we could decide in the future not to use ids, but subselect, so this is more robust.

Reply:
@kgaikwad @ezr-ondrej Done! make them use "unscoped" as suggested.